SQL/01_Tabloid_Create_DB.sql

```diff
@@ -57,7 +57,9 @@ CREATE TABLE [Subscription] (
 
 CREATE TABLE [Category] (
   [Id] integer PRIMARY KEY IDENTITY,
-  [Name] nvarchar(50) NOT NULL
+  [Name] nvarchar(50) NOT NULL,
+
+  CONSTRAINT [UQ_Category_Name] UNIQUE([Name])
```

Comment:
Rather than changing your create script, I think it would be better to leave the original and add an alter script to the branch so that your teammates can run it when they pull main. You can put it in the SQL folder, and start the file name with `03` to indicate it's to be run after the others.